cluster/manager.go

```diff
@@ -1516,6 +1516,11 @@ func (c *ClusterManager) NodeRemoveDone(nodeID string, result error) {
 			nodeID, err)
 		logrus.Errorf(msg)
 	}
+
```

Comment:
can we move the code before deleteNodeFromDB ? this way even if the node crashes after remove config we can still re-run decommission again ?

Reply:
done